Formula/gmt.rb

```diff
@@ -30,9 +30,9 @@ class Gmt < Formula
   end
 
   resource "dcw" do
-    url "https://github.com/GenericMappingTools/dcw-gmt/releases/download/2.1.0/dcw-gmt-2.1.0.tar.gz"
-    mirror "https://mirrors.ustc.edu.cn/gmt/dcw-gmt-2.1.0.tar.gz"
-    sha256 "e810567cd474d5af9d84751938acb447a8612391ed66190d60d789974a1f9e29"
+    url "https://github.com/GenericMappingTools/dcw-gmt/releases/download/2.1.1/dcw-gmt-2.1.1.tar.gz"
+    mirror "https://mirrors.ustc.edu.cn/gmt/dcw-gmt-2.1.1.tar.gz"
+    sha256 "ab72c8253076380c8477a1225370bc41f4aff811c28d2f64d75052ff3d9003af"
```

Comment:
CI is failing because this checksum doesn't match the one from the downloaded file (`d4e208dca88fbf42cba1bb440fbd96ea2f932185c86001f327ed0c7b65d27af1`).

Reply:
Thanks, I've fixed the checksum.